test/e2e/infinispan/smoke_test.go

```diff
@@ -76,6 +77,32 @@ func TestBaseFunctionality(t *testing.T) {
 	}
 }
 
+// Test if the ConfigMap is applying the custom logging configuration
+func TestCustomLogging(t *testing.T) {
+	defer testKube.CleanNamespaceAndLogOnPanic(t, tutils.Namespace)
+
+	// Infinispan cluster definition with extra labels to be propagated to the service and pods
```

Comment:
Comment does not apply to this test can be removed.

Reply:
ops